.changes/next-release/Bug Fix-8f3202fe-3dbc-47dd-bb57-3a384de226ad.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "introduce time threshold for intellisense type trigger to avoid chain reaction trigger"
```

Comment:
I don't think this change is user-facing

Reply:
okk will not add it then